mix.exs

```diff
@@ -57,7 +57,7 @@ defmodule Astarte.VMQ.Plugin.Mixfile do
     ]
   end
 
-  defp elixirc_paths(:test), do: ["lib", "test/support"]
+  defp elixirc_paths(:test), do: ["test/support", "lib"]
```

Comment:
Since the order is important here and you had to manually swap it, add a comment underlining this to avoid someone accidentally swapping this again